packages/core/admin/ee/server/services/review-workflows/workflows.js

```diff
@@ -11,8 +13,19 @@ module.exports = ({ strapi }) => ({
     return strapi.entityService.findOne(WORKFLOW_MODEL_UID, id, opts);
   },
 
-  create(workflowData) {
-    return strapi.entityService.create(WORKFLOW_MODEL_UID, { data: workflowData });
+  async create(opts) {
+    let createOpts = opts;
+
+    // Create stages if provided
+    if (opts.data.stages) {
```

Comment:
I think stages should always be provided, because we don't allow empty workflows.